src/main/webapp/app/quiz/participate/quiz.module.ts

```diff
@@ -30,8 +28,7 @@ const ENTITY_STATES = [...quizRoute];
         DragAndDropQuestionComponent,
         QuizScoringInfoStudentModalComponent,
         ShortAnswerQuestionComponent,
-        DragItemComponent,
-        SecuredImageComponent
+        DragItemComponent
```

Comment:
Is there a reason why you have removed SecuredImageComponent here? Does the quiz module still work without it?

Reply:
I added it to the shared module. it is imported here as well